testing/tests/DevExpress.ui.widgets/list.markup.tests.js

```diff
@@ -371,6 +371,21 @@ QUnit.module('decorators markup', {}, () => {
         assert.ok($deleteToggle.find(toSelector(TOGGLE_DELETE_SWITCH_ICON_CLASS)).length, 'toggle icon generated');
     });
 
+    QUnit.test('list item markup, toggle delete decorator is visible with showSelection mode: true', function(assert) {
```

Comment:
```suggestion
    QUnit.test('list item delete icon is visible when showSelectionControls=true', function(assert) {
```

isn't it more clear title?